product_list_price_from_pricelist/models/res_company.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    base_pricelist_compute_price_id = fields.Many2one(
+        "product.pricelist",
+        string="Pricelist to compute sale price",
```

Comment:
Remember to always use shorter labels in fields, and put longer explanations in help. Adjust now accordingly the settings view, maybe putting a specific label for that view that allows longer texts.

```suggestion
        string="Recomputing pricelist",
        help="Pricelist for computing the product sales price",
```